### PR TITLE
fix midi out for newly created projects

### PR DESCRIPTION
**Pull request by maks**

### CODE REVIEW

**Comment by maks** [comment]
@democloid ok I think it should be ready for review now.

File: sources/Services/Midi/MidiService.cpp [hunk 1]
```diff
@@ -146,9 +144,14 @@ void MidiService::stopDevice() {
 void MidiService::OnPlayerStart() {
 
   if (deviceName_.size() != 0) {
+    // printf("ReStart Midi DEVICE NAME:%s", deviceName_.c_str());
     stopDevice();
     startDevice();
-    deviceName_ = "";
+    deviceName_ = "MIDI OUT 1";
```
Comment (democloid) [inline sources/Services/Midi/MidiService.cpp:150]: shouldn't device name be set up before start? I think original code clears it after start to leave it blank for next run, but may be mistaken.

**Comment by RAWJUNGLE** [comment]
@democloid this fix causes a bug with the instruments

**Comment by democloid** [comment]
> @democloid this fix causes a bug with the instruments

@RAWJUNGLE you'll have to explain what the bug is, I'm not an mind reader

**Comment by RAWJUNGLE** [comment]
@democloid  For example, if I create a chain, a phrase, put an instrument and click to select another instrument I get a midi channel at once, that is, I go I00 to I01, a midi channel appears, and if I want to switch back somehow, the tracker freezes, and so on and so forth. 

It doesn't matter if it's an old project or a new one, it's useless.

**Comment by maks** [comment]
outdated

**Pull request closed without merging**